src_features/performPrivacyOperation/ui_common_performPrivacyOperation.c

```diff
@@ -0,0 +1,27 @@
+#include "shared_context.h"
+#include "feature_getPublicKey.h"
+#include "ui_callbacks.h"
+
+unsigned int io_seproxyhal_touch_privacy_ok(__attribute__((unused)) const bagl_element_t *e) {
+    uint32_t tx = set_result_perform_privacy_operation();
+    G_io_apdu_buffer[tx++] = 0x90;
+    G_io_apdu_buffer[tx++] = 0x00;
```

Comment:
why `0x90` and `0x00`

Reply:
see previous comment for status code 0x9000